feat: env docs (#3234)
kubeshop/testkube: docs/docs/concepts/tests/tests-creating.md

When you run such a test you will face a memory limit for the scraper pod, when the default scraper job template doesn't have any resource constraints.

> ### **Mounting config maps and secrets to executor pod**

Reviewer: ```suggestion
### Mounting ConfigMap and Secret to executor pod
```

Reviewer: prefer no ** ** 

prefer Kubernees concepts to be capitalized

> If you need to mount your config maps and secrets to your executor environment, then you can provide them as additional  

Reviewer: ```suggestion
If you need to mount your ConfigMap and Secret to your executor environment, then you can provide them as additional  
```

parameters when you create or run the test using `--mount-configmap` and `--mount-secret` options:

> ```bash
> kubectl testkube create test --file test/postman/LocalHealth.postman_collection.json --name mount-test --type postman/collection --mount-configmap your_configmap=/pod_mount_path --mount-secret your_secret=/pod_mount_path
> ```

Reviewer: ```suggestion
```sh
testkube create test --file test/postman/LocalHealth.postman_collection.json --name mount-test --type postman/collection --mount-configmap your_configmap=/pod_mount_path --mount-secret your_secret=/pod_mount_path
```
```

Reviewer: prefer sh over bash

prefer not using `kubectl testkube`, but using `testkube`

> ### **Automatically add all config map and secret keys to test variables**

Reviewer: ```suggestion
### Automatically add all ConfigMap and Secret keys to Test Variables
```

Reviewer: Prefer using Test Variables capitalized, as it's a core Testkube concept

Sometimes you may want to automatcially add all keys from your config map and secret to your test. In this you need to provide them as additional  

> parameters when you create or run the test using `--variable-configmap` and `--variable-secret` options and they will be automatically added during test execution:

Reviewer: ```suggestion
Sometimes you may want to automatcially add all keys from your ConfigMap and Secret to your test. In this you need to provide them as additional  
parameters when you create or run the test using `--variable-configmap` and `--variable-secret` options and they will be automatically added during test execution:
```

> ```bash
> kubectl testkube create test --file test/postman/LocalHealth.postman_collection.json --name var-test --type postman/collection --variable-configmap your_configmap --variable-secret your_secret
> ```

Reviewer: ```suggestion
```sh
testkube create test --file test/postman/LocalHealth.postman_collection.json --name var-test --type postman/collection --variable-configmap your_configmap --variable-secret your_secret
```
```
